CHANGELOG.md

```diff
@@ -7,14 +7,19 @@ breaking changes should be noted here. It should be more concise than `git log`.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-[Unreleased]
+[Unreleased][Unreleased_log]
 ------------
 
+[v0.8.0][v0.8.0_log] - 2020-01-22
+---------------------
+
 ### Added
 
 - Support for backtracing in debug and release builds.
     - Implementations for GNU functions `backtrace` and `backtrace_symbols` (defined in execinfo.h)
     - Enclaves are built using `-fno-omit-frame-pointer` for accurate backtraces.
+- Support for custom attestation data formats via new plugin model. Please refer to [design documentation](docs/DesignDocs/CustomAttestation.md).
```

Comment:
Grammar (missing article): "Please refer to _the_ design documentation."

Can we file an issue to track ensuring that the doc actually reflects the current implementation if we are going to use it as public documentation? A quick scan shows that some of the type names have changed at least (e.g. `oe_attester_plugin_t` is now `oe_attestation_role_t`)

Reply:
Fixed. thanks.

Reply:
Opened issue #2458